ion_functions/data/adcp_functions.py

```diff
@@ -859,50 +901,90 @@ def adcp_beam2ins(b1, b2, b3, b4):
 
         2013-04-10: Christopher Wingard. Initial code.
         2015-06-24: Russell Desiderio. Incorporated int fillvalue -> Nan.
+        2019-08-13: Christopher Wingard. Adds functionality to compute a 3-beam solution
+                    and cleans up syntax used in the function.
 
     Usage:
 
-        u, v, w, e = adcp_beam2ins(b1, b2, b3, b4)
+        x, y, z, e = adcp_beam2ins(b1, b2, b3, b4, pg1, pg2, pg3, pg4)
 
             where
 
-        u = "east" velocity profiles in instrument coordinates [mm s-1]
-        v = "north" velocity profiles in instrument coordinates [mm s-1]
-        w = "vertical" velocity profiles in instrument coordinates [mm s-1]
-        e = "error" velocity profiles [mm s-1]
+        x = x axis velocity profiles in instrument coordinates [mm s-1]
```

Comment:
Could we retain some indication that our x axis here is "east-west" and our y axis "north-south" and the z axis "vertical"? While that's the intuitive interpretation, it seems better to explicitly identify what the coordinates mean.

Reply:
No, that is incorrect. At this stage, the x, y, z axis are relative to the instrument and require a further transformation (utilizing the compass, tilt/roll sensors, and the instrument orientation) to align them with the east-west/north-south axis. This function is specifically to transform the data from the individual beam coordinates to instrument coordinates.